frontend_server/lib/server.dart

```diff
@@ -45,7 +47,14 @@ ArgParser _argParser = new ArgParser(allowTrailingOptions: true)
           ' Intended use is to satisfy different loading strategies implemented'
           ' by gen_snapshot(which needs platform embedded) vs'
           ' Flutter engine(which does not)',
-      defaultsTo: true);
+      defaultsTo: true)
+  ..addFlag('track-widget-creation-locations',
+      help: 'Run a kernel transformer to track creation locations for widgets '
+      'When in batch mode, link platform kernel file into result kernel file.'
+      ' Intended use is to satisfy different loading strategies implemented'
+      ' by gen_snapshot(which needs platform embedded) vs'
+      ' Flutter engine(which does not)',
```

Comment:
trivial nits: spaces before `(`, also, put the space at the end of the line rather than the start, also, line up the strings so their open `'`s are all vertically aligned.